geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/help/ScalaConsoleCommand.scala

```diff
@@ -6,21 +6,23 @@
  * http://www.opensource.org/licenses/apache2.0.php.
  ***********************************************************************/
 
-package org.locationtech.geomesa.tools.status
+package org.locationtech.geomesa.tools.help
```

Comment:
Hmm... 'help' (and 'status') both seem like odd package names for the Scala Console command.  (I'm not opposed just mentioning it in case there's an obvious better option after some reflection.)

Separately, I wonder if there are split packages here that we will need to think about.

Reply:
`help` _is_ kind of confusing haha... these are all the "placeholder" commands that are actually implemented in the bash script, but we want them to show up in the jcommander help. if you've got a suggestion i'd be happy to change it.

re: split packages, i'm not going to worry about that until we need to, but you're probably right.

Reply:
Yeah, that is great for now.   We'll figure it out when we sort out Java 11 support;).